<launch>
    <include file="$(find kelsey_sandbox)/launch/kinect.launch" />

    <node name="nano25" 
		pkg="netft_rdt_driver" 
		type="netft_node" 
		args="192.168.2.120" 
		respawn="true">
    <remap from="/netft_data" to="/tool_netft/wrench_raw"/>
    </node>

    <node name="adl2" 
		pkg="ros_vrpn_client"
		type="ros_vrpn_client"
		args="_vrpn_server_ip:=192.168.2.110"
	        respawn="false" output="screen"/>

<!--
    <node name="pr2_antenna" 
		pkg="ros_vrpn_client"
		type="ros_vrpn_client"
		args="_vrpn_server_ip:=192.168.2.110"
	        respawn="false" output="screen"/>
-->

    <node name="head" 
		pkg="ros_vrpn_client"
		type="ros_vrpn_client"
		args="_vrpn_server_ip:=192.168.2.110"
	        respawn="false" output="screen"/>

    <node name="head_center"
        pkg="tf"
        type="static_transform_publisher"
        args="0.21 0.02 -0.087 0 0 0 /head /head_center 10" />

    <node name="kinect" 
		pkg="ros_vrpn_client"
		type="ros_vrpn_client"
		args="_vrpn_server_ip:=192.168.2.110"
	        respawn="false" output="screen"/>

    <node name="kinect_calibration_board" 
		pkg="ros_vrpn_client"
		type="ros_vrpn_client"
		args="_vrpn_server_ip:=192.168.2.110"
	        respawn="false" output="screen"/>

   <node name="kinect_cb_corner"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.0327 0.03146 -0.00563 0 0 0 /kinect_calibration_board /kinect_cb_corner 10" />

    <node name="scratcher"
        pkg="tf" 
        type="static_transform_publisher"  
        args="0.072 0 0 0 0 0 /tool_netft_frame /scratcher 10" />

    <node name="shaver"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.070 0 0 0 0 0 /tool_netft_frame /shaver 10" />

   <node name="comb"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.179 0 -0.02 0 0 0 /tool_netft_frame /comb 10" />

   <node name="tooth_brush"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.147 0 -0.018 0 0 0 /tool_netft_frame /tooth_brush 10" />

   <node name="wipe" 
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.137 0 0 0 0 0 /tool_netft_frame /wipe 10" />

   <node name="wipe_finger"
       	pkg="tf" 
       	type="static_transform_publisher" 
       	args="0.094 0 0 0 0 0 /tool_netft_frame /wipe_finger 10" />

   <node name="spoon"
        pkg="tf" 
        type="static_transform_publisher" 
        args="0.115 0 0 0 0 0 /tool_netft_frame /spoon 10" />

    <node name="tool_netft_frame"
		pkg="tf" 
		type="static_transform_publisher"  
		args="0.234 0.041 -0.015 0 0 0 /adl2 /tool_netft_frame 10" />

    <node name="tool_netft_raw_frame"
		pkg="tf" 
		type="static_transform_publisher"  
		args="0.234 0.041 -0.015 0.5 -0.5 0.5 -0.5 /adl2 /tool_netft_raw_frame 10" />

   <node name="kinect_calib"
		pkg="tf" 
		type="static_transform_publisher" 
		args="0.089455 -0.002889 -0.057416 -0.481324 0.501655 -0.512041 0.504463 /kinect /openni_rgb_optical_frame 10" />
    <!-- 8/31 kinect calibration error 0.0101 mean error out of N=100 samples -->

</launch>
